recipes/libsecret/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "libsecret" %}
+{% set version = "0.18.6" %}
+{% set sha256 = "5efbc890ba41a323ffe0599cd260fd12bd8eb62a04aa1bd1b2762575d253d66f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://ftp.gnome.org/pub/gnome/sources/libsecret/0.18/libsecret-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+  patches:
+    - 0001-Disable-tests-which-require-additional-deps.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+  run_exports:
+    #  https://abi-laboratory.pro/tracker/timeline/libsecret/
+    - {{ pin_subpackage('libsecret', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - intltool
+    - pkg-config
+    - sed  # [osx]
+    # intltool hard pins perl, can't rely on cbc.yaml here
+    - perl 5.22.2.1
+    - python 2.7
+  host:
+    - glib
+    - libgcrypt
```

Comment:
These don't have `run_exports` yet. Could we have a `run` requirement with them listed?

Reply:
Aren't we relying on the `pin_run_as_build` thingy?

https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/b64714b82a0a6d17d0f63022118be965f3935530/recipe/conda_build_config.yaml#L95-L96

and

libgcrypt has `run_exports` https://github.com/conda-forge/libgcrypt-feedstock/blob/master/recipe/meta.yaml#L16-L17

Reply:
If you're using `pin_run_as_build`, you still need it listed in `run`; [example](https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level).